frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -349,6 +397,32 @@ function getMetricListItem(
   };
 }
 
+function getCompareListItem(
```

Comment:
We could fix `XXXListItem` in this component and add a `type` property and make it a proper union instead.

Reply:
51e0023ea8be4d32fdd34afad9b3c4ae1a6a8ebb